Release-Notes/Release1.md

```diff
@@ -17,19 +17,25 @@ the container layer with k8s Cluster API (now in version 0.4.x).
 
 ## Metrics collection and dashboards
 
-### Prometheus exporters
+### Prometheus exporters and Grafana dashboards
 
-### Grafana Dashboards
+We provide generic configuration examples and blueprints for prometheus rules and grafana dashboards. The examples need to be understood and adapted to the particular needs of your environment. You can find the examples at <https://github.com/osism/kolla-operations>. 
+
+With R2 we plan to implement a basic set of these alerts and dashboards in the testbed deployment in order to make them even easier consumable for new users. Feel free to give feedback on the examples and contribute your own generic examples. 
+
+We're working on bringing a basic set of prometheus exporters to the OpenStack-kolla upstream community. 
+
+As part of our effort to add further monitoring tooling, we're integrating further prometheus exporters such as [libvirt](https://review.opendev.org/c/openstack/kolla-ansible/i+/643568) and [ovn](https://review.opendev.org/c/openstack/kolla/+/762986). Integration is targeted for R2.
 
 ### openstack-health-monitor
 
 ### ...
 
-
 ## Logging
 
 ### Central logging
 
+OSISM now enables kolla-ansible centralized logging by default. The default rules need to be further refined to suit your needs. We plan to implement a more generic set of rules for R2.
```

Comment:
@berendt We weren't absolutely sure in the SIG Monitoring call today whether or not logging is now enabled by default in OSISM. Can you please confirm this?